internal/gatewayapi/route.go

```diff
@@ -390,16 +390,23 @@ func (t *Translator) processGRPCRouteRule(grpcRoute *GRPCRouteContext, ruleIdx i
 		}
 
 		if match.Method != nil {
-			if match.Method.Method != nil {
+			// GRPC's path is in the form of "/<service>/<method>"
+			// TODO: support regex match type after https://github.com/kubernetes-sigs/gateway-api/issues/1746 is resolved
```

Comment:
can you raise a GH issue within EG so its easier for this project to track it

Reply:
https://github.com/envoyproxy/gateway/issues/1071